### PR TITLE
added patatorostisslides into the selected recipe

### PR DESCRIPTION
**Pull request by Eliasadam**

### CODE REVIEW

File: src/components/PotatoRostisSlides.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  Carousel,
+  CarouselItem,
+  CarouselControl,
+  CarouselIndicators,
+  CarouselCaption
+} from 'reactstrap';
+
+
+
+class PotatoRostisSlides extends Component {
```
Comment (Samatar26) [inline src/components/PotatoRostisSlides.js]: @Eliasadam You have three slides components for three different recipes. The only difference in these components is the data they're using. So you can just have one component called recipeslides and that just take in different data/recipes 👍

File: src/components/ChickPeaWelcomePage.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
```
Comment (Samatar26) [inline src/components/ChickPeaWelcomePage.js]: @Eliasadam Similar to the three recipe slides component. You should have just one WelcomePage component that renders the correct recipeslides component depending on what the user chose in the Recipepreparation component. 👍

File: src/components/pages/DashboardPage.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import { Button } from 'reactstrap';
 import TableSetup from '../TableSetup';
 import RecipePreparation from '../RecipePreparation';
+//import RecipeSlides from '../RecipeSlides';
```
Comment (Samatar26) [inline src/components/pages/DashboardPage.js]: As the component isn't being used, you can remove this comment.

File: src/data/chickPeaSaladSlides.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "altText": " Step 1: Peel and finely slice the red onion. Add to bowl. ",
```
Comment (Samatar26) [inline src/data/chickPeaSaladSlides.json]: Great job on the data for the recipesteps 👍 💪

File: src/components/PotatoRostisSlides.js (hunk 1 above)
Comment (Eliasadam) [reply src/components/PotatoRostisSlides.js]: yes thats right. ok I will thank you for reviewing my code

**Pull request merged**